docker-compose.yml

```diff
@@ -10,9 +10,7 @@ services:
       POSTGRES_USER: lrsql_user
```

Comment:
I am not sure that we want to make ALL uses of docker-compose be clustered. I would consider dumping this one into dev-resources.